### PR TITLE
Added support for unzipped .dcm/.IMA files

### PR DESCRIPTION
**Pull request by dpark6060**
Successful run: 
https://ss.ce.flywheel.io/#/jobslog/job/5fbbe1c296f335b9df35f11a

### CODE REVIEW

File: manifest.json [hunk 1]
```diff
@@ -8,7 +8,7 @@
   "source": "https://github.com/flywheel-apps/extract-cmrr-physio/releases",
   "license": "MIT",
   "flywheel": "0",
-  "version": "1.2.3",
+  "version": "1.2.4",
   "custom": {
     "docker-image": "flywheel/extract-cmrr-physio:1.2.3",
```
Comment (AndysWorth) [inline manifest.json]: Doesn't this need to be bumped to 1.2.4 also?  And also for just below this?

File: run.py [hunk 2]
```diff
@@ -135,25 +135,38 @@ def extract_zipped_dicom(gear_context, dicom):
...
+        
+        if is_zip:
+            zip_base = op.splitext(op.split(dicom)[-1])[0]
+    
+            # Now we need to unzip it:
+            uz_dir = op.join('/tmp', 'unzipped_dicom')
+            unzip_dicom_command = ['unzip', '-o', dicom, '-d', uz_dir]
+            ###########################################################################
+            # Try to run the unzip command:
+            exec_command(gear_context, unzip_dicom_command)
```
Comment (AndysWorth) [inline run.py:148]: Could do this in python maybe

**Review by AndysWorth — APPROVED** [review]
LGTM!

File: manifest.json (hunk 1 above)
Comment (dpark6060) [reply manifest.json]: fixed

**Pull request merged**